src/input/plugins/CurlInputPlugin.cxx

```diff
@@ -83,7 +83,7 @@ struct CurlInputStream final : public AsyncInputStream, CurlResponseHandler {
 	CurlInputStream(const CurlInputStream &) = delete;
 	CurlInputStream &operator=(const CurlInputStream &) = delete;
 
-	static InputStream *Open(const char *url, Mutex &mutex, Cond &cond);
+	static InputStream *Open(const char *url, Mutex &mutex, Cond &cond, Tag *tag);
```

Comment:
What's this new parameter? Add API documentation. Why is it writable? Who owns the pointer?

Reply:
This was a hack to apply metadata to a `CurlInputStream` - I don't expect it to stay.